css/counter.css

```diff
@@ -0,0 +1,59 @@
+:root {
+    --light-blue: rgb(223, 235, 250);
+    --medium-blue: rgb(139, 186, 247);
+    --dark-blue: rgb(16, 74, 175)
+}
+
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+main {
+    display: grid;
+    place-items: center;
+    padding-top: 8rem;
+}
+
+.counter {
+    border-radius: 1rem;
+    background-color: var(--light-blue);
+    padding: 0.5rem 1rem;
+    width: 15rem;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.counter__input {
+    width: 4rem;
+    height: 4rem;
+    border: 2px solid var(--medium-blue);
+    border-radius: 1rem;
+    font-size: 2rem;
+    font-weight: 700;
+    color: var(--dark-blue);
+    text-align: center;
+}
+
+.counter__input:focus-visible {
```

Comment:
`:focus-visible` 😎 not obvious choice but the correct one 😄

Reply:
Thanks, I've spent some time to discover it! 😄 That is the last example of the element's states in DevTools xd